README.md

```diff
@@ -1,219 +1,41 @@
 # PocketIC: A Canister Testing Platform
 
-PocketIC is a local testing platform for canisters on the [Internet Computer](https://internetcomputer.org/).
+PocketIC is a local canister testing platform for the [Internet Computer](https://internetcomputer.org/).  
+It consists of the PocketIC **binary**, which handles the different IC instances, and a **Python library** (this repo) which allows you to interact with your canisters and the IC itself.
 
-
-Using PocketIC requires two components: A server and an integration library. 
-
-The **server** is compiled as part of the build process for the Internet Computer. It runs as local HTTP server which manages IC instances. Every test may request its own, _independent_ IC instance and interact with it concurrently with other running tests. 
-
-This PocketIC **library** provides a convenient Python frontend to interact with IC instances, while hiding the REST-API layer, the process lifetime etc. The test author simply interacts with either an IC instance:
-
-```python
```

Comment:
How about about keeping both code examples; but shortening the prose text?

Reply:
Deal ;)